fuse_models/src/graph_ignition.cpp

```diff
@@ -180,13 +188,24 @@ void GraphIgnition::process(const fuse_msgs::msg::SerializedGraph& msg)
     }
 
     auto srv = std::make_shared<std_srvs::srv::Empty::Request>();
-    // No need to spin since node is optimizer node, which should be spinning
-    auto result_future = reset_client_->async_send_request(srv);
-    result_future.wait();
+    // Don't block the executor.
+    // It needs to be free to handle the response to this service call.
+    // Have a callback do the rest of the work when a response comes.
+    auto result_future = reset_client_->async_send_request(srv,
+      [this, post_process, &graph, msg](rclcpp::Client<std_srvs::srv::Empty>::SharedFuture result){
```

Comment:
- I had to convert the graph uniqueptr into a shared_ptr so it can be copyable.
	- This is because the async service call signature we're using requires a std::function, which must be copyable (and so requires all args to be copyable)

But aside from that, this bug is fixed! Nice catch!